library/core/src/primitive_docs.rs

```diff
@@ -1564,6 +1564,36 @@ mod prim_ref {}
 ///
 /// In addition, all *safe* function pointers implement [`Fn`], [`FnMut`], and [`FnOnce`], because
 /// these traits are specially known to the compiler.
+///
+/// All function pointers implement the trait [`FnPtr`], which implements the following traits:
+///
+/// * [`ConstParamTy`]
```

Comment:
We should probably leave the unstable traits out of this list, no?

Reply:
and then should I delete the dummy impl for `FnPtr` and add one for `Sized`?

Reply:
hey @thomcc, just following up on this